packages/wallet/frontend/src/components/icons/Eye.tsx

```diff
@@ -0,0 +1,39 @@
+import { SVGProps } from 'react'
+
+type EyeProps = SVGProps<SVGSVGElement> & { loading?: boolean }
+
+export const Eye = ({ loading, ...props }: EyeProps) => {
```

Comment:
What do we use loading for? I didn't see the svg called with this parameter

Reply:
removed unused prop